packages/web/components/Guild/Section/GuildPlayers.tsx

```diff
@@ -50,7 +50,7 @@ export const GuildPlayers: React.FC<Props> = ({ guildId, guildname }) => {
     <ProfileSection title="Players">
       {loadingPlayers && <LoadingState />}
 
-      {guildPlayers == null ? (
+      {guildPlayers == null || guildPlayers.length === 0 ? (
```

Comment:
This shows up when `loadingPlayers` is true.  We want to show only loading state initially and then show either "no known players yet" or the list of players.

Reply:
ahh yeah, gotcha. will fix that.